docs/user-guide/structured.ipynb

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
<p>Suggest adding the link to the dataset descriptions in case people are interested? <a href="https://docs.xarray.dev/en/stable/generated/xarray.tutorial.open_dataset.html" target="_blank">https://docs.xarray.dev/en/stable/generated/xarray.tutorial.open_dataset.html</a></p><p>and perhaps showcasing the structured grid dataarray first for comparison with the unstructured grid dataset below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1040/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Good idea!